2-mandatory/3-financial-times.js

```diff
@@ -6,6 +6,7 @@
 */
 function potentialHeadlines(allArticleTitles) {
     // TODO
+   return allArticleTitles.filter(items => items.length < 65);
```

Comment:
The instruction asks for `65 characters or less`

Reply:
fixed 👍